index.html

```diff
@@ -379,11 +556,34 @@ <h5>SUBSCRIBE</h5>
                         <li><a href="">Digital Subscriptions</a></li>
                         <li><a href="">Games</a></li>
                         <li><a href="">Cookings</a></li>
+                        <li class="list_sep"><a href="">Election 2020</a></li>
+                     
+                        <li><a href="">Cookings</a></li>
+                        <li><a href="">Election 2020</a></li>
+                        <li><a href="">Cookings</a></li>
+                        <li><a href="">Election 2020</a></li>
+                        <li><a href="">Cookings</a></li>
                         <li><a href="">Election 2020</a></li>
```

Comment:
There are a tags with empty href. Remember to use a hash(#) symbol in between the quotes, if there is no specific file path to include.